buildspecs/j9.flags

```diff
@@ -1003,14 +1003,14 @@ all data is stored in their own region. It additionaly reduces footprint, mainly
 		<description>DEBUGGING FEATURE.  Determines if the interpreter produces debug information at every bytecode.</description>
 		<ifRemoved>No full trace available.</ifRemoved>
 	</flag>
-	<flag id="interp_updateVMCTracing">
-		<description>Debug mode for tracing uses of updateVMStruct</description>
-		<ifRemoved>No tracing of updateVMStruct helpers</ifRemoved>
-	</flag>
 	<flag id="interp_twoPassExclusive">
 		<description>Exclusive VM access - Set halt bit in one pass, count responders in another pass</description>
 		<ifRemoved>Set and count in a single pass</ifRemoved>
 	</flag>
+	<flag id="interp_updateVMCTracing">
+		<description>Debug mode for tracing uses of updateVMStruct</description>
+		<ifRemoved>No tracing of updateVMStruct helpers</ifRemoved>
+	</flag>
```

Comment:
This change looks to be gratuitous.  Unintentional?

Reply:
Its to make the flags be in alphabetical order. Makes my life easier so that order between the uma flags and the cmake flags stays the same

Reply:
That makes sense.  I'm fine with including it then